examples/angular/src/pages/ui/components/authenticator/custom-slots/custom-slots.component.html

```diff
@@ -18,7 +18,7 @@
     </h3>
   </ng-template>
 
-  <ng-template amplifySlot="sign-in-footer">
+  <ng-template amplifySlot="sign-in-footer" let-authenticator>
```

Comment:
I like this, but this doesn't match the dx of React and Vue. At least for the examples. In React and Vue we receive the `useAuthenticator/authenticator` function by importing it in, but maybe it's OK doing it either way? 

Is it better dx to expose it through `let-*` or importing the service in is the question? Or maybe both is fine? @ericclemmons and I were discussing this yesterday. I don't have a strong opinion either way. If this is the way we want to go though, should I then expose authenticator through Vue's scoped slots too...

Reply:
Good point. For Angular, I do really enjoy this DX, though I see that other framework used `useAuthenticator` syntax. Maybe it's okay for Angular as it's more template-oriented than Vue? 

My idea is that template variables should be first class citizen, then AuthenticatorServices.

Reply:
For now, I think I want to leave it at `let-authenticator`. After #665 I can look to document `AuthenticatorService` better.

Reply:
OK, 👍